server/core/storelimit/limit.go

```diff
@@ -22,16 +22,30 @@ const (
 	AddPeer Type = iota
 	// RemovePeer indicates the type of store limit that limits the removing peer rate
 	RemovePeer
+	SendSnap
```

Comment:
Where do we use it?

Reply:
I has no place needs to use it in past, I will remove it and add it if neccessary.